core/src/main/java/org/mule/runtime/core/internal/processor/LoggerMessageProcessor.java

```diff
@@ -36,8 +36,10 @@
  */
 public class LoggerMessageProcessor extends AbstractComponent implements Processor, Initialisable, MuleContextAware {
 
+  // TODO - MULE-16446: Logger execution type should be defined according to the appender used
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7658&issues=AWjKMxB-otIXnmsUXcbg&open=AWjKMxB-otIXnmsUXcbg)

[ISSUE_KEY]: AWjKMxB-otIXnmsUXcbg